oam/install_scripts/columnstore-post-install.in

```diff
@@ -310,16 +310,14 @@ chmod 777 /dev/shm
 
 systemctl cat mariadb-columnstore.service > /dev/null 2>&1
 if [ $? -eq 0 ] && [ $(running_systemd) -eq 0 ]; then
-    # prevent clusters using shared storage from initializing columnstore more than once
-    IFLAG=/var/lib/columnstore/storagemanager/cs-initialized
     mkdir -p /var/lib/columnstore/storagemanager
     chown -R mysql:mysql /var/lib/columnstore/storagemanager
 
-    if [ ! -e $IFLAG ]; then
-       touch $IFLAG
-       echo "Populating the engine initial system catalog."
-       systemctl start mariadb-columnstore
-    fi
+    echo "Populating the engine initial system catalog."
+
+    # prevent nodes using shared storage manager from stepping on each other when initializing
+    flock /var/lib/columnstore/storagemanager -c "systemctl start mariadb-columnstore"
+    systemctl stop mariadb-columnstore # shutdown after initialization
```

Comment:
This does three things. It keeps columnstore consistent on both docker and systemd environments, it keeps columnstore consistent with mariadb install, but most importantly it allows user to configure storagemanager.cnf after installation without hitting error on savebrm when columnstore is reloaded after modifying config (i.e., loaded localstorage, but attempted to savebrm without a storagemanager process.

Reply:
Verified our official docs have user starting up mariadb-columnstore via systemctl start mariadb-columnstore so this does not cause any issues on user side

Reply:
Could you share the docs that you refer to? I'm not aware about installation guide for community version.

Reply:
https://mdbcdt.com/DOCS-2085/deploy/community-single-columnstore/#post-installation